Month2/week1/Month2-Project Week 1-Remembering the Bases/VideoGame/VideoGames.cpp

```diff
@@ -0,0 +1,75 @@
+#include "VideoGames.h"
+
+FVideoGame::FVideoGame()
+{
+	GameName = "";
```

Comment:
Keep in mind that classes with default constructors don't need to be explicitly initialized, as their constructors are called upon definition. So, there's really no need to explicitly initialize GameName or StudioName :)

Please fix this for the rest of your review

This is _only_ for class instances with default constructors. The way you are assigning the date fields is fine since they are not initialized by default.